### PR TITLE
Docker Keycloak

### PR DESCRIPTION
**Pull request by latch2112**

### CODE REVIEW

File: Keycloak/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    restart: always
+    depends_on:
+      - postgres
+    ports:
+      - 8443:8443
+    environment:
+      KEYCLOAK_USER: "leAdmin"
+      KEYCLOAK_PASSWORD: ${kcpassword}
+      DB_VENDOR: "postgres"
+      DB_PASSWORD: ${ppassword}
```
Comment (latch2112) [inline Keycloak/docker-compose.yml]: Keycloak images does not support secret for DB password

Comment (nyanofthemoon) [reply Keycloak/docker-compose.yml]: Noooooooo~

**Review by nyanofthemoon — COMMENTED** [review]
@latch2112 Usually we put an empty `.gitkeep` file instead of `Keycloak/postgres/volumeForPostgres.md`

File: Keycloak/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  keycloak:
+    image: jboss/keycloak:4.8.3.Final
+    restart: always
+    depends_on:
+      - postgres
+    ports:
+      - 8443:8443
+    environment:
+      KEYCLOAK_USER: "leAdmin"
```
Comment (nyanofthemoon) [inline Keycloak/docker-compose.yml]: Can't we just use `admin`...

File: Keycloak/docker-compose.yml (hunk 1 above)
Comment (nyanofthemoon) [inline Keycloak/docker-compose.yml]: Could we use `POSTGRESQL_PASSWORD` instead? :)

File: Keycloak/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      KEYCLOAK_PASSWORD: ${kcpassword}
```
Comment (nyanofthemoon) [inline Keycloak/docker-compose.yml]: Could we use `KEYCLOAK_PASSWORD` instead? :)

File: Keycloak/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_USER: "keycloak"
+      POSTGRES_PASSWORD: ${ppassword}
+      POSTGRES_DB: "keycloak"
+    volumes:
+      - ./postgres:/var/lib/postgresql/data
+    ports:
+      - 5432:5432
```
Comment (nyanofthemoon) [inline Keycloak/docker-compose.yml]: Maybe we don't need to expose it since we'll only be calling keycloak service itself.

**Review by nyanofthemoon — CHANGES_REQUESTED** [review]
Please! 🦄

**Review by nyanofthemoon — APPROVED** [review]
LGTM!

**Pull request merged**